app/assets/javascripts/services/miq_service.js

```diff
@@ -158,4 +158,20 @@ ManageIQ.angular.app.service('miqService', ['$timeout', '$document', '$q', 'API'
       miqService.sparkleOff();
     }
   };
+
+  this.redirectBack = function(message, flashType, redirectUrl, reload) {
+    var flash = { message: message };
+
+    flash.level = flashType;
+
+    miqFlashLater(flash);
+
+    $timeout(function() {
+      $window.location.href = redirectUrl;
+
+      if(reload) {
```

Comment:
Pretty sure I was seeing the issue before.
And that timeout block was very much needed before. In fact, I had to adjust the `500 milli seconds` value based on a few tests.

But don't see the issue anymore. Not sure what changed.